cli/default-reporter/test/index.ts

```diff
@@ -298,6 +298,9 @@ test('prints summary without the filtered out entries', (done) => {
       expect(output).toBe(EOL + `\
 ${h1('dependencies:')}
 ${ADD} foo ${versionColor('1.0.0')} ${versionColor('(2.0.0 is available)')}
+` + EOL + `\
```

Comment:
I'm not sure why this test kept giving an error on my branch, but after this change, both the main branch and the current branch workes fine.

It seems to actually print that a dev dependency qar has been added ?